apps/api/src/api/application-api.ts

```diff
@@ -57,3 +63,39 @@ export const editApplication = async ({ id, update }: { id: number; update: Appl
 		return failure(message);
 	}
 };
+
+/**
+ *
+ * @param userId - user ID
+ * @param state - application state
+ * @param sort - sorting options
+ * @param page - page offset
+ * @param pageSize - page limit
+ * @returns Success with list of Applications / Failure with Error
+ */
+export const getAllApplications = async ({
+	userId,
+	state,
+	sort,
+	page,
+	pageSize,
+}: {
+	userId: string;
+	state?: ApplicationStates;
+	sort?: Array<OrderBy<ApplicationsColumnName>>;
+	page?: number;
+	pageSize?: number;
+}) => {
+	const database = getDbInstance();
+	const service: ApplicationService = applicationService(database);
+
+	const result = await service.listApplications({ user_id: userId, state, sort, page, pageSize });
+
+	if (!result.success) {
+		const message = `Cannot return applications list`;
+		console.error(message);
+		return failure(message);
+	}
```

Comment:
I think you can just return `result` here because if it's a failure in `service.listApplications` we will want to return the error message as well.